modules/RTC/MungeLocalSdp.js

```diff
@@ -0,0 +1,343 @@
+
+                // Aborts the forEach on this particular track,
+                // but will continue with the other ones
+                // eslint-disable-next-line no-continue
+                continue;
+            }
+
+            // FIXME come up with a message
+            // when there should not be audio SSRC anymore
+            if (audioMLine.getSSRCCount() > 0) {
+                logger.debug(
+                    'Doing nothing - audio SSRCs are still there');
+
+                // audio SSRCs are still there
+                // eslint-disable-next-line no-continue
+                continue;
+            }
+
+            modified = true;
+
+            // We need to fake sendrecv
+            audioMLine.direction = 'sendrecv';
+
+            logger.debug(`Injecting audio SSRC: ${audioSSRC}`);
+            audioMLine.addSSRCAttribute({
+                id: audioSSRC,
+                attribute: 'cname',
+                value: `injected-${audioSSRC}`
+            });
+            audioMLine.addSSRCAttribute({
+                id: audioSSRC,
+                attribute: 'msid',
+                value: audioTrack.storedMSID
+            });
+        }
+
+        return modified;
+    }
+
+    /**
+     * Makes sure local SDP for video media reflects current local tracks status
+     * of the parent TPC.
+     *
+     * NOTE 1 video track is assumed
+     *
+     * @param {SdpTransformWrap} transformer the transformer instance which will
+     * be used to process the SDP.
+     * @return {boolean} <tt>true</tt> if any modification were made.
+     * @private
+     */
+    _describeLocalVideoTracks(transformer) {
+
+        // Go over each video tracks and check if the SDP has to be changed
+        const localVideos = this.pc.getLocalTracks(MediaType.VIDEO);
+
+        if (!localVideos.length) {
+            return false;
+        } else if (localVideos.length !== 1) {
+            logger.error(
+                'There is more than 1 video track ! '
+                    + 'Strange things may happen !', localVideos);
+        }
+
+        const videoMLine = transformer.selectMedia('video');
+
+        if (!videoMLine) {
+            logger.error(
+                'Unable to hack local video track SDP - no "video" media');
+
+            return false;
+        }
+
+        if (videoMLine.direction === 'inactive') {
+            logger.error(
+                `Not doing local video transform for direction: ${
+                    videoMLine.direction}`);
+
+            return false;
+        }
+
+        let modified = false;
+
+        for (const videoTrack of localVideos) {
+            const isMuted = videoTrack.isMuted();
+            const muteInProgress = videoTrack.inMuteOrUnmuteProgress;
+            const isAttached = videoTrack._isAttachedToPC(this.pc);
+            const shouldFakeSdp = isMuted || muteInProgress || !isAttached;
+
+            logger.debug(
+                `${videoTrack
+                 } isMuted: ${isMuted
```

Comment:
I agree, but we used that as a convention for wrapping template string instead of using + sign. I'd rather keep it like this until we decide on strict rule for this case. There are many other places like this not only in this project.